### ISSUE TITLE
NameError: name 'Image' is not defined

### ISSUE THREAD

**Issue by sajith** [atlanticwave-sdx/sdx-lc#26]
`./swagger_server/messaging/message_queue_producer.py` has a `__main__`, but it can't be run:

```
python3 ./swagger_server/messaging/message_queue_producer.py 
Traceback (most recent call last):
  File "[..]./swagger_server/messaging/message_queue_producer.py", line 59, in <module>
    image = Image(filename="local-ctlr-1.manifest")
NameError: name 'Image' is not defined
```

**Comment by sajith** [comment]
Where does `Image` come from, and why does it attempt to load a JSON file?

https://github.com/atlanticwave-sdx/sdx-lc/blob/6cb6e2d7c46a2e0f7e66814912e8f85faeb8abdf/swagger_server/messaging/message_queue_producer.py#L59-L60

**Comment by sajith** [comment]
We can remove the usage of `Image` with this:

```python
    f = open(file="./mq-test/local-ctlr-1/local-ctlr-1.manifest")
    data = json.load(f)
```

But then another problem arises:

```
python3 ./swagger_server/messaging/message_queue_producer.py 
Traceback (most recent call last):
  File "[..]./swagger_server/messaging/message_queue_producer.py", line 64, in <module>
    with RabbitMq(server) as rabbitmq:
AttributeError: __enter__
```

Seems that these lines:

https://github.com/atlanticwave-sdx/sdx-lc/blob/6cb6e2d7c46a2e0f7e66814912e8f85faeb8abdf/swagger_server/messaging/message_queue_producer.py#L62-L63

.. could just be:

```python
    RabbitMq(server).publish(body=data)
```

And with that we should be able to run `python3 ./swagger_server/messaging/message_queue_producer.py`, if RabbitMQ is running:

```console
python3 ./swagger_server/messaging/message_queue_producer.py 
Published Message: {'endpoints': [...]}
```

PR will follow.

**Comment by sajith** [comment]
This is also related to #14.

**Comment by sajith** [comment]
Fixed with PR #32.
